src/scripts.js

```diff
@@ -80,6 +81,25 @@ function displayFilteredTag(){
     };
 };
 
+function displayFilteredFavorite() {
+    const tagSelected = checkTagType();
+    const favList = user.recipesToCook
+    const tagSelectedList = user.filterToCookByTag(tagSelected)
+   
+    favoritesView.innerHTML = ""
+
+    if(tagSelected === "all"){
```

Comment:
Kiko and Jenn, I know that you are aware of this but I'm wondering if it would be better to add a reset button instead of an 'all'? And then have the recipes filter on the button click of the radio button instead of having a submit button.